lib/Alchemy/Phrasea/SearchEngine/Elastic/Search/FacetsResponse.php

```diff
@@ -31,6 +31,16 @@ public function __construct(Escaper $escaper, array $response)
         }
     }
 
+    /**
+     * Term aggregates
+     *
+     * @return array
+     */
+    public function getFacets()
+    {
+        return $this->facets;
```

Comment:
You should not do that. This is private structure, if you expose it we will not be able to change it anymore.